SQL/GenerateData.fsx

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
So I want enough data for this to be compelling, so I'm going to use some code for autogenerating data with a normal distribution.  Does anybody have a problem with me using F# for this?  This isn't something the user would be expected to need to use or understand, it's just how I'm generating the SQL statements in PopulateDemoData.sql.

Also, this might make for a really ugly demo because the observations won't look anything like the forecasts.  Maybe I want something where I sort of define the distribution for each month individually?  But that might be overkill

Reply:
This is no longer true after planning.  This will be deleted and we'll write a separate project to populate observations using the API